signals/content/subject_suspicious_keywords.yml

```diff
@@ -0,0 +1,195 @@
+    "account.*de-?activat",
+    "account.*locked",
+    "account.*re-verification",
+    "account.*security",
+    "account.*suspension",
+    "account.has.been",
+    "account.has.expired",
+    "account.will.be.blocked",
+    "account v[il]o[li]at",
+    "activity.*acc(oun)?t",
+    "almost.full",
+    "app[li]e.[il]d",
+    "authenticate.*account",
+    "been.*suspend",
+    "clos.*of.*account.*processed",
+    "confirm.your.account",
+    "courier.*able",
+    "deactivation.*in.*progress",
+    "delivery.*attempt.*failed",
+    "document.received",
+    "documented.*shared.*with.*you",
+    "dropbox.*document",
+    "e-?ma[il1]+ .{010}suspen",
+    "e-?ma[il1]{1} user",
+    "e-?ma[il1]{2} acc",
+    "e-?ma[il1]{2}.*up.?grade",
+    "e.?ma[il1]{2}.*server",
+    "e.?ma[il1]{2}.*suspend",
+    "email.update",
+    "faxed you",
+    "fraud(ulent)?.*charge",
+    "from.helpdesk",
+    "fu[il1]{2}.*ma[il1]+[ -]?box",
+    "has.been.*suspended",
+    "has.been.limited",
+    "have.locked",
+    "he[li]p ?desk upgrade",
+    "heipdesk",
+    "i[il]iega[il]",
+    "ii[il]ega[il]",
+    "incoming e?mail",
+    "incoming.*fax",
+    "lock.*security",
+    "ma[il1]{1}[ -]?box.*quo",
+    "ma[il1]{2}[ -]?box.*fu[il1]",
+    "ma[il1]{2}box.*[il1]{2}mit",
+    "ma[il1]{2}box stor",
+    "mail on.?hold",
+    "mail.*box.*migration",
+    "mail.*de-?activat",
+    "mail.update.required",
+    "mails.*pending",
+    "messages.*pending",
+    "missed.*shipping.*notification",
+    "missed.shipment.notification",
+    "must.update.your.account",
+    "new [sl][io]g?[nig][ -]?in from",
+    "new voice ?-?mail",
+    "notifications.*pending",
+    "office.*3.*6.*5.*suspend",
+    "office365",
+    "on google docs with you",
+    "online doc",
+    "password.*compromised",
+    "periodic maintenance",
+    "potential(ly)? unauthorized",
+    "refund not approved",
+    "revised.*policy",
+    "scam",
+    "scanned.?invoice",
+    "secured?.update",
+    "security breach",
+    "securlty",
+    "signed.*delivery",
+    "status of your .{314}? ?delivery",
+    "susp[il1]+c[il1]+ous.*act[il1]+v[il1]+ty",
+    "suspicious.*sign.*[io]n",
+    "suspicious.activit",
+    "temporar(il)?y deactivate",
+    "temporar[il1]{2}y disab[li]ed",
+    "temporarily.*lock",
+    "un-?usua[li].activity",
+    "unable.*deliver",
+    "unauthorized.*activit",
+    "unauthorized.device",
+    "undelivered message",
+    "unread.*doc",
+    "unusual.activity",
+    "upgrade.*account",
+    "upgrade.notice",
+    "urgent message",
+    "urgent.verification",
+    "v[il1]o[li1]at[il1]on security",
+    "va[il1]{1}date.*ma[il1]{2}[ -]?box",
+    "verification ?-?require",
+    "verification( )?-?need",
+    "verify.your?.account",
+    "web ?-?ma[il1]{2}",
+    "web[ -]?ma[il1]{2}",
+    "will.be.suspended",
+    "your (customer )?account .as",
+    "your.office.365",
+    "your.online.access",
+
+    // https://github.com/sublime-security/static-files/blob/master/suspicious_subjects.txt
```

Comment:
we could use $suspicious_subjects now for the non-regex ones

Reply:
Great call